core/vm/evm_test.go

```diff
@@ -21,3 +29,27 @@ func TestAffectedMode_Update_whenTypical(t *testing.T) {
 		}
 	}
 }
+
+func TestCall_shouldReturnErrorNotAuthorized_whenNoCodeAndSupportsMultitenancy(t *testing.T) {
```

Comment:
@nicolae-leonte-go @nmvalera @trung please review the UT for that situation mentioned above

Reply:
LGTM